### ISSUE TITLE
'Delete/Remove' Github data source in service view fails

### ISSUE THREAD

**Issue by saralavanip** [Digipalvelutehdas/dipor-dashboard#369]
# 'Delete/Remove' Github data source in service view fails
    User unable to remove/delete github source in service view

# Steps to reproduce  

1. Visit https://dashboard.digipalvelutehdas.fi

2. Login

3. Create Organization

4. Go to Organization view and select the organization added above

5. Create Department

6. Go to department view

7. Add 'Service'  

8. Go to department view and select the service added above       

9. Click on 'Add Source'    

10. Go to source and delete source added above  
  
# Observed behavior
   Delete github data source in service view fails

# Expected behavior  
   User must be able to delete github data source in service view  

- Web browser: 
  -  Mozilla Firefox 44.0
  -  Chrome 48.0.2564.103

- Operating system  
  - Linux : Ubuntu 14.04.03 LTS (Trusty)

**Comment by frenchbread** [comment]
Getting same error locally
```
Exception while invoking method '/sources/remove' ReferenceError: source is not defined
```

**Comment by frenchbread** [comment]
Created hotfix #372.

**Comment by saralavanip** [comment]
Verified the resolved issue in latest deployed version 0.6.0 https://dashboard.digipalvelutehdas.fi
